### PR TITLE
fix(recording): call toggleRecording with no args

### PR DESCRIPTION
**Pull request by paweldomas**

### CODE REVIEW

File: modules/xmpp/recording.js [hunk 1]
```diff
@@ -334,10 +334,15 @@ Recording.prototype.setRecording = function(...args) {
...
+ * @param {Object} options
+ * @param {string} options.token token for authentication
+ * @param {string} options.streamId the stream ID to be used with Jibri in
+ * the streaming mode.
  * @param statusChangeHandler {function} receives the new status as argument.
  */
-Recording.prototype.toggleRecording = function(options, statusChangeHandler) {
+Recording.prototype.toggleRecording = function(
+    options = { },
+    statusChangeHandler) {
```
Comment (saghul) [inline modules/xmpp/recording.js]: I see `statusChangeHandler` is used unconditionally. Do we really support calling this function without args, or is it just working out of luck?

Comment (paweldomas) [reply modules/xmpp/recording.js]: Sorry, it should say calling JitsiConference.toggleRecording with no args

Comment (saghul) [reply modules/xmpp/recording.js]: Then how are we calling it? `conference.toggleRecording(undefined, statusChangeHandler);` ?

Comment (saghul) [reply modules/xmpp/recording.js]: If so, LGTM.

Comment (paweldomas) [reply modules/xmpp/recording.js]: We're calling it like this:
https://github.com/jitsi/lib-jitsi-meet/blob/master/JitsiConference.js#L1696
The purpose of this PR is to allow to pass undefined options.

**Pull request merged**